lessons/beginners/install/windows.md

```diff
@@ -1,6 +1,9 @@
 # Instalace Pythonu pro Windows
 
-Běž na [stahovací stránku Pythonu](https://www.python.org/downloads/)
+Pokud máš verzi Windows 10, napiš do příkazové řádky python nebo python3, 
```

Comment:
```suggestion
Pokud máš verzi Windows 10, zadej do příkazové řádky příkaz `python`.
```

* Není potřeba zmiňovat obě možnosti.
* Je to samostatná věta.
* napiš + *zmáčkni Enter* = zadej příkaz